05_silly_blocks/silly_blocks.rb

```diff
@@ -0,0 +1,17 @@
+def reverser
+  yield.split.map { |word| word.reverse }.join(" ")
+end
+
+def adder(x = 1)
+  yield + x
+end
+
+def repeater(x = 0, &block)
+  if x == 0
```

Comment:
Please stop doing exceptional cases -- it's error prone, and both branches do basically the same thing :) Can you include this in the iteration? 

```
    (1..x).each do |var|
      yield
    end
```